Update daikin doc (#24093)
home-assistant/home-assistant.github.io: source/_integrations/daikin.markdown

## Sensor

The `daikin` sensor platform integrates Daikin air conditioning systems into Home Assistant, enabling displaying the following parameters by device:

- Inside temperature
- Inside humidity
- Hourly energy consumption in cool mode
- Hourly energy consumption in heat mode
- Today's total energy consumption (updated hourly, resets at 00:00)

> The integration displays the following parameters for the outdoor compressor:

Reviewer: The outdoor measurements were removed from the PR, right?

Author: The "outdoor" device has been removed but not the sensors, and IMO it's still worth distinguishing indoor and outdoor sensors in the doc

Reviewer: Ok.

<div class='note'>

- The 'Outdoor compressor Energy consumption' and 'Outdoor compressor Estimated power consumption' sensors are updated every time 100 Wh are consumed by all different operating modes summed together.
- The 'Outdoor compressor Estimated power consumption' sensor is derived from the above energy consumption and not provided by the AC directly.
- The 'cool/heat' energy sensors are updated hourly with the previous hour energy consumption
  of a given mode and a given AC.
- The 'cool' mode also includes the 'fan' and 'dehumidifier' modes' power consumption.

> - Would you have multiple indoor devices, the 'Outdoor compressor' sensors will be created multiple times but will all report the same values. You should disable all but one.

Reviewer: ```suggestion
- If you have multiple indoor devices, the 'Outdoor compressor' sensors will be created multiple times but will all report the same values. You can disable all but one.
```
